en/flight_modes_vtol/return.md

```diff
@@ -0,0 +1,67 @@
+# Return Mode (VTOL)
+
+[<img src="../../assets/site/position_fixed.svg" title="Position fix required (e.g. GPS)" width="30px" />](../getting_started/flight_modes.md#key_position_fixed)
+
+The _Return_ flight mode is used to _fly a vehicle to safety_ on an unobstructed path to a safe destination, where it may either wait (hover or circle) or land.
+
+VTOL vehicles use the [Mission Landing/Rally Point](../flight_modes/return.md#mission-landing-rally-point-return-type-rtl-type-1) return type by default.
+In this return type a vehicle ascends to a safe altitude above obstructions (if needed), and then flies directly to a rally point or the start of a mission landing point (whichever is nearest), or the home position if neither rally points or mission landing pattern is defined.
+If the destination is a mission landing pattern, the vehicle will then follow the pattern to land.
+If the destination is a rally point or the home location, the vehicle will fly back to the home position and land as a multicopter.
+
+VTOL supports the [other PX4 return types](../flight_modes/return.md#return-types-rtl-type), including home/rally point return, mission path and closest safe destination.
+The default type is recommended.
+
+:::note
+
+- Mode is automatic - no user intervention is _required_ to control the vehicle.
+- Mode requires a global 3d position estimate (from GPS or inferred from a [local position](../ros/external_position_estimation.md#enabling-auto-modes-with-a-local-position)).
+  - Flying vehicles can't switch to this mode without global position.
+  - Flying vehicles will failsafe if they lose the position estimate.
+- Mode requires home position is set.
+- Mode prevents arming (vehicle must be armed when switching to this mode).
+- RC control switches can be used to change flight modes on any vehicle.
+- RC stick movement is ignored.
+
+<!-- https://github.com/PX4/PX4-Autopilot/blob/main/src/modules/commander/ModeUtil/mode_requirements.cpp -->
+
+:::
+
+## Technical Summary
+
+VTOL vehicles use the [Mission Landing/Rally Point](../flight_modes/return.md#mission-landing-rally-point-return-type-rtl-type-1) return type by default.
+In this return type the vehicle:
+
```

Comment:
I would separate it into two parts: RTL sequence when triggered while in hovering state, and while in forward flying state. And for the hovering it maybe is enough to reference to the MC docs, as it's 1:1 the same.

Reply:
Done.